dlr/dlr-engine/src/main/java/ai/djl/dlr/engine/DlrModel.java

```diff
@@ -63,8 +63,19 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
     }
 
     private void checkModelFiles(String prefix) throws IOException {
+        String libExt;
+        String os = System.getProperty("os.name").toLowerCase();
+        if (os.startsWith("mac")) {
+            libExt = ".dylib";
+        } else if (os.startsWith("linux")) {
+            libExt = ".so";
+        } else if (os.startsWith("win")) {
+            libExt = ".dll";
+        } else {
+            throw new IllegalStateException("found unsupported os");
+        }
         // TODO make the check platform independent
-        Path module = modelDir.resolve(prefix + ".dylib");
+        Path module = modelDir.resolve(prefix + libExt);
```

Comment:
```suggestion
        Path module = modelDir.resolve(System.mapLibraryName(prefix));
```

Reply:
System.mapLibraryName will include "lib"  prefix. The model name doesn't start with lib usually